salt/utils/parsers.py

```diff
@@ -610,9 +610,10 @@ def _mixin_setup(self):
             *self._console_log_level_cli_flags,
             dest=self._loglevel_config_setting_name_,
             choices=list(salt._logging.LOG_LEVELS),
-            help="Console logging log level. One of {}. Default: '{}'.".format(
+            help="Console logging log level. One of {}. Default: '{}'. \n The following log levels are INSECURE and may log sensitive data: {}".format(
                 ", ".join(["'{}'".format(n) for n in salt._logging.SORTED_LEVEL_NAMES]),
                 self._default_logging_level_,
+                ", ".join(insecure_log()),
```

Comment:
```suggestion
            help=(
                "Console logging log level. One of {}. Default: '{}'.\n"
                "The following log levels are INSECURE and may log sensitive data: {}"
            ).format(
                ", ".join(["'{}'".format(n) for n in salt._logging.SORTED_LEVEL_NAMES]),
                self._default_logging_level_,
                ", ".join(insecure_log()),
```
Minor formatting change for smaller line length, but, is the added line break actually respected?

Reply:
> So yes, its respected. It looks cleaner to me to have that line break, but its not required if you want to remove it.

If it's respected, leave it.